tests/tests-html-elements.php

```diff
@@ -197,4 +197,41 @@ public function test_month_dropdown() {
 		$this->assertContains( '<option value="11"', $out );
 		$this->assertContains( '<option value="12"', $out );
 	}
+
+	public function test_select_is_required() {
```

Comment:
These are great, but for each one can you add a docblock like this:

```php
/**
 * @covers EDD_HTML_Elements::select
 */
```

For the one that covers text, you'd change `select` to `text` (the `::select` references the method name in the HTML class).